controllers/akodeploymentconfig/cluster/cluster_controller_unit_test.go

```diff
@@ -45,6 +45,8 @@ loadBalancerAndIngressService:
             subnet_ip: 10.0.0.0
             subnet_prefix: "24"
             network_name: test-akdc
+            control_plane_network_name: test-akdc-cp
```

Comment:
what happens if the info for `control_plane_network` is not provided? would it be assigned to some default values or be the same with data network? Personally I would like a unit test for this case

Reply:
we have this logic here: https://github.com/vmware-samples/load-balancer-operator-for-kubernetes/blob/lxinqi/avi-network-separation/pkg/ako/values.go#L263